### ISSUE TITLE
Loading of message list appears slow due to spinner on avatars

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#2727]
When you open the Mail app and things load in, actually the last thing still appearing to load are the avatar previews with big spinners on all of them:
![avatars loading just show letter placeholders](https://user-images.githubusercontent.com/925062/76059623-eab80500-5fb1-11ea-89c3-9b2de92fabe3.png)

This makes the performance appear unnecessarily slow, as even the first message is already displayed when that spinning still happens. It even seems as if you should wait until that is properly loaded.

Instead, we should just _not_ show the spinners as the avatars are non-essential. That’s exactly what we have the letter placeholders for. :) And once the avatars are loaded, we can quickly fade them in there, replacing the letter placeholder. And if an avatar doesn’t load – that’s totally fine. :)

**Comment by ahstro** [comment]
This seems to be an upstream "issue", in [Nextcloud's Vue components](https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Avatar/Avatar.vue#L41). Don't feel entirely comfortable overwriting that on an app level? Maybe this should be discussed higher up?

**Comment by ChristophWurst** [comment]
Agreed. This is something that can and should be fixed there. Either this is configurable or we make it the default to show the textual fallback as long as the image is loading.
